### PR TITLE
Condense gardenctl commands layout

### PR DESCRIPTION
**Pull request by petersutter**
**What this PR does / why we need it**:
We got the feedback that the gardenctl commands occupy too much space.
The commands are now moved to the Access card and only shown when clicking on the view icon.

Screenshot 2020-04-23 at 17 39 42

Screenshot 2020-04-23 at 17 40 03

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
`gardenctl` commands are now shown under the `Access` card
```

### CODE REVIEW

File: frontend/src/components/ShootDetails/GardenctlCommand.vue [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      expansionPanel: false
+    }
+  },
+  computed: {
+    visibilityIcon () {
+      if (this.expansionPanel) {
+        return 'visibility_off'
+      } else {
+        return 'visibility'
+      }
```
Comment (holgerkoser) [inline frontend/src/components/ShootDetails/GardenctlCommand.vue:88]:
```suggestion
        return this.expansionPanel ? 'visibility_off' : 'visibility'
```

File: frontend/src/components/ShootDetails/GardenctlCommand.vue [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    },
+    visibilityTitle () {
+      if (this.expansionPanel) {
+        return 'Hide Command'
+      } else {
+        return 'Show Command'
+      }
```
Comment (holgerkoser) [inline frontend/src/components/ShootDetails/GardenctlCommand.vue:91]:
```suggestion
      return this.expansionPanel ? 'Hide Command' : 'Show Command'
```

File: frontend/src/components/ShootDetails/GardenctlCommand.vue [hunk 3]
```diff
@@ -0,0 +1,109 @@
+<!--
```
Comment (holgerkoser) [inline frontend/src/components/ShootDetails/GardenctlCommand.vue:1]: Wy do we need this component?

File: frontend/src/components/ShootDetails/GardenctlCommands.vue [hunk 4]
```diff
@@ -15,44 +15,27 @@ limitations under the License.
...
-        </v-list-item-content>
-        <v-list-item-action>
-          <copy-btn :clipboard-text="value"></copy-btn>
-        </v-list-item-action>
-      </v-list-item>
-    </v-list>
-  </v-card>
+  <div>
+    <div v-for="({ title, subtitle, value }, index) in commands" :key="title">
+      <gardenctl-command
```
Comment (holgerkoser) [inline frontend/src/components/ShootDetails/GardenctlCommands.vue:20]: inline code from `gardenctl-command`. To many unnecessary div's .

File: frontend/src/components/ShootDetails/GardenctlCommands.vue [hunk 5]
```diff
@@ -15,44 +15,27 @@ limitations under the License.
...
-            ></code-block>
-          </v-list-item-title>
-        </v-list-item-content>
-        <v-list-item-action>
-          <copy-btn :clipboard-text="value"></copy-btn>
-        </v-list-item-action>
-      </v-list-item>
-    </v-list>
-  </v-card>
+  <div>
```
Comment (holgerkoser) [inline frontend/src/components/ShootDetails/GardenctlCommands.vue:18]: wrap with v-list

File: frontend/src/components/ShootDetails/GardenctlCommand.vue [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+    <v-list-item v-if="expansionPanel">
+      <v-list-item-icon></v-list-item-icon>
+      <v-list-item-content>
+        <code-block
+          lang="shell"
+          :content="'$ ' + value.replace(/ --/g, ' \\\n    --')"
+          :show-copy-button="false"
+        ></code-block>
+      </v-list-item-content>
+    </v-list-item>
```
Comment (holgerkoser) [inline frontend/src/components/ShootDetails/GardenctlCommand.vue:50]:
```suggestion
  <v-expand-transition>
    <v-card flat>
        <code-block
          lang="shell"
          :content="'$ ' + value.replace(/ --/g, ' \\\n    --')"
          :show-copy-button="false"
        ></code-block>
      </v-card>
    </v-expand-transition>
```

File: frontend/src/components/ShootDetails/ShootAccessCard.vue [hunk 7]
```diff
@@ -129,6 +129,10 @@ limitations under the License.
         <code-block lang="yaml" :content="shootInfo.kubeconfig" :show-copy-button="false"></code-block>
```
Comment (holgerkoser) [inline frontend/src/components/ShootDetails/ShootAccessCard.vue:129]: remove negative margin from last expansion panel

File: frontend/src/components/ShootDetails/GardenctlCommands.vue (hunk 4 above)
Comment (petersutter) [reply frontend/src/components/ShootDetails/GardenctlCommands.vue:20]: ✅

File: frontend/src/components/ShootDetails/GardenctlCommands.vue (hunk 5 above)
Comment (petersutter) [reply frontend/src/components/ShootDetails/GardenctlCommands.vue:18]: ✅

File: frontend/src/components/ShootDetails/ShootAccessCard.vue (hunk 7 above)
Comment (petersutter) [reply frontend/src/components/ShootDetails/ShootAccessCard.vue:129]: ✅

File: frontend/src/components/ShootDetails/GardenctlCommand.vue (hunk 3 above)
Comment (petersutter) [reply frontend/src/components/ShootDetails/GardenctlCommand.vue]: ✅

File: frontend/src/components/ShootDetails/GardenctlCommand.vue (hunk 6 above)
Comment (petersutter) [reply frontend/src/components/ShootDetails/GardenctlCommand.vue]: In my opinion it looks better if we use the list-item-content layout (having the padding on the left), also the expand transition does not seem to work (even if I add the v-if to the v-card)

Comment (holgerkoser) [reply frontend/src/components/ShootDetails/GardenctlCommand.vue]: Yes. You are right.

**Pull request merged**